lib/arrays.md

```diff
@@ -5,44 +5,44 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is an Array?
 
-Replace me with your response...
+An array is an ordered collection of objects. Each object in an array is given an integer index value, beginning with 0. Arrays are similar to strings but can contain many more types of objects, such as numbers, strings, and other arrays.
 
 # What are some examples of information that would be Arrays as opposed to some other data type?
 
-Replace me with your response...
+A list of non-consecutive numbers would contained in an array, as would a combination of fixnums and floats. A list of strings would also be an array. 
 
 # What is one way, using Ruby, to retrieve the 6th element in an Array? How about the 8th element? What happens if you try to retrieve the value of the _9999th_ element (or some element that doesn't exist in the array)?
 
-Replace me with your response...
+Each element in an array is assigned an index number, starting with 0, and an element can be retrieved by using that index. For example, if an_array = ["blue", "red", "yellow", "green", "purple", "orange", "black", "white"], then print an_array[5] will return "orange". The same would be true for the 8th element, print an_array[7] would return "white". Because there is no 50th element, calling print an_array[50] will return nil.
 
 # The previous question asks about finding, for example, the 6th element in an Array. Is it possible to find the **-6th** (Notice the negative symbol!) element in an Array? What does that even mean?
 
-Replace me with your response...
+It is possible to find a negative index in an array. If a negative index is accessed, the indexing order is reversed and the last value is assigned 0 rather than the first. So calling print an_array[-5] would return "yellow".
```

Comment:
Close! When moving backward through an Array, indices start at `-1` (that way there can't be two elements at index `0`). Calling`an_array[-5]` would return `"green"` and `an_array[-6]` would return `"yellow"`.